pylint/checkers/format.py

```diff
@@ -969,11 +969,12 @@ def check_lines(self, lines, i):
         max_chars = self.config.max_line_length
         ignore_long_line = self.config.ignore_long_lines
 
-        for line in lines.splitlines(True):
+        for line in splitlines(lines, True):
```

Comment:
Let's pass ``True`` as a keyword argument, as in ``keepends=True``. We can't know when reading the code what ``True`` will stand for, so a name will help the comprehension of the code.